README.md

```diff
@@ -1 +1,34 @@
-# Git Assignment - YuriyKoshulap
\ No newline at end of file
+a. What is an issue?
+An Issue in GitHub is a bug tracker for a project which has its own section in every repository.
+
+b. What is a pull request?
+A pull request is a proposal to merge a set of changes from one branch into another.
+
+c. How do I open up a pull request?
+
+git pull origin branch-name
+
+or
+
+git pull origin main
```

Comment:
No.. to open a pull request you can do so with the GitHub GUI, not pulling updates from remote branch